tools/cert_create/Makefile

```diff
@@ -27,6 +28,12 @@ MAKE_HELPERS_DIRECTORY := ../../make_helpers/
 include ${MAKE_HELPERS_DIRECTORY}build_macros.mk
 include ${MAKE_HELPERS_DIRECTORY}build_env.mk
 
+ifeq (${USE_TBBR_DEFS},1)
```

Comment:
minor: It would be good to add a comment here saying that cert_tool is platform independent in this case